package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "webapis",
+  "private": "true",
+  "namespace": "@arcstack",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ArcStackLab/webapis.git"
+  },
+  "packageManager": "pnpm@9.2.0",
+  "description": "A Comprehensive Integration of All Web Standard APIs",
+  "scripts": {
+    "lint": "prettier . --check && eslint",
+    "format": "prettier . --write",
+    "uninstall-husky": "pnpm uninstall husky --no-save && git config --unset core.hooksPath && pnpm dlx rimraf .husky"
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "pnpm dlx lerna version && git add .",
```

Comment:
The pre-push hook uses `lerna version`, which can be a heavy operation. It might lead to long wait times during push operations.

```diff
-      "pre-push": "pnpm dlx lerna version && git add .",
+      "pre-push": "pnpm dlx lerna version --no-git-tag-version && git add .",
```